send/atlassian_mu

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+SERVICE_NAME="atlassian_mu"
+EXECSCRIPT="./atlassian_mu_process.pl"
+
+FACILITY_NAME=$1
+DESTINATION=$2
+DESTINATION_TYPE=$3
+
+	#Test if destination is in not empty and in the correct format
+if [ -z "$DESTINATION" ]; then
+	echo "Missing Destination argument (Name of configuration file to retrieve endpoint URL and key from)" >&2
+	exit 252
+else
+	if echo "$DESTINATION" | grep "[^.A-Za-z_0-9-]"; then
+		echo "Bad format of destination!" >&2
+		exit 251
+	fi
+fi
+
+#Test if name of facility is not empty
+if [ -z "$FACILITY_NAME" ]; then
+	echo "Missing FacilityName argument" >&2
+	exit 245
+fi
+
+#Destination is name of instance (name of file with other information like authorization etc.)
+INSTANCE_NAME=$DESTINATION
```

Comment:
I think we can skip this and use $DESTINATION on the only place where the instance name has been used.